Analysis/config/2018/training_inputs_MC.cfg

```diff
@@ -0,0 +1,7 @@
+DY: dir=(DY.*) file=eventTuple.* types=jet,e,mu,tau
+TT: dir=(TT.*) file=eventTuple.* types=jet,e,mu,tau
+W:  dir=(W1Jets.*|W2Jets.*|W3Jets.*|W4Jets.*|WJets.*) file=eventTuple.* types=jet,e,mu,tau
+QCD: dir=(QCD.*) file=eventTuple.* types=jet
+Higgs: dir=(ttHTo.*|GluGluHTo.*|ZHTo.*|WplusHTo.*|WminusHTo.*|VBF.*|GluGlu.*) file=eventTuple.*  types=jet,e,mu,tau
```

Comment:
- from Higgs samples, I suggest taking only tau
- Add embedded samples (taking emb_tau and emb_jet)